subprojects/docs/src/snippets/jvmComponents/scala/groovy/build.gradle

```diff
@@ -1,29 +0,0 @@
-/*
```

Comment:
❌ This snippet is referenced in `SampleScalaLanguageIntegrationTest`

Reply:
Which I removed, since the snippet is not used in any documentation and this is the legacy software model.

Reply:
I've read each change except for the last one :) All good. And big thumbs-up for removing old/unnecessary code.